docs/sql-ref-syntax-qry-select-groupby.md

```diff
@@ -113,7 +116,18 @@ aggregate_name ( [ DISTINCT ] expression [ , ... ] ) [ FILTER ( WHERE boolean_ex
         (warehouse, location),
         (warehouse, size),
         (warehouse))`.
- 
+    
+    `GROUP BY warehouse, GROUPING SETS((env), GROUPING SETS((product), ()), GROUPING SETS((location, size), (location), (size), ()))`
```

Comment:
Can we give an explanation on the behavior instead of examples only? We have a good explaination for the mixed case:
```
For multiple `GROUPING SETS` in the `GROUP BY` clause, we generate
a single `GROUPING SETS` by doing a cross-product of the original `GROUPING SETS`s.
```

Can we do the same for nested case?

Reply:
how about current?